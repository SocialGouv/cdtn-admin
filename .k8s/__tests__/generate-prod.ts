//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);
test("kosko generate --prod", async () => {
  expect(
    await getEnvManifests("prod", "", {
      ...project("cdtn-admin").prod,
      IP_ALLOWLIST: "123.456.456.789,42.0.0.0/8",
    })
  ).toMatchSnapshot();
});
