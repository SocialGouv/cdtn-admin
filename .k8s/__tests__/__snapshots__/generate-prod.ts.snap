// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate --prod 1`] = `
"42
"
`;
